ChineseFilter/sections/specific.txt

```diff
@@ -44,6 +44,9 @@ people.com.cn##.ad_left
 microsoftnews.msn.cn##div[class^="nativeAd-DS-card"]
 ||assets.msn.cn/bundles/v1/microsoftNews/latest/placement-manager.*.js$domain=microsoftnews.msn.cn
 zcool.com.cn##.main-wrapper-advlong-top
+tvbs.com.tw###news_detail_div > div[align="center"]
+tvbs.com.tw###news_detail_div > strong > a[rel="noreferrer"]
+tvbs.com.tw###popular_rank > li:not([id])
```

Comment:
dangerous. It is not big
![image](https://user-images.githubusercontent.com/8361299/110124217-7d100600-7dca-11eb-9f38-730e1702d224.png)